infra/README.md

```diff
@@ -10,26 +10,26 @@ This CLI tool simplifies the management and setup of local environments for deve
 
 ### Available Commands
 
-While the CLI offers various commands, one of the most powerful and useful for developers is the `run-local-env` command. This command allows developers to render the required environment and start a local development environment with the necessary configuration.
+While the infra tool offers various commands, one of the most powerful and useful for developers is the `run-local-env` command. This command allows developers to render the required environment and start a local development environment with the necessary configuration.
 
 Here is a summary of all available commands in the CLI:
 
 ```text
-yarn cli <command>
+yarn infra <command>
 
 Commands:
-  yarn cli render-env        Render a chart for environment
-  yarn cli render-urls       Render urls from ingress for environment
-  yarn cli render-secrets    Render secrets secrets needed by service
-  yarn cli render-env-vars   Render environment variables needed by service.
+  yarn infra render-env        Render a chart for environment
+  yarn infra render-urls       Render urls from ingress for environment
+  yarn infra render-secrets    Render secrets secrets needed by service
+  yarn infra render-env-vars   Render environment variables needed by service.
                              This is to be used when developing locally and
                              loading of the environment variables for "dev"
                              environment is needed.
-  yarn cli render-local-env  Render environment variables needed by service.
+  yarn infra render-local-env  Render environment variables needed by service.
                              This is to be used when developing locally and
                              loading of the environment variables for "dev"
                              environment is needed.
-  yarn cli run-local-env     Render environment and run the local environment.
+  yarn infra run-local-env     Render environment and run the local environment.
```

Comment:
Spacing needs to be adjusted

Reply:
Maybe just delete this and instruct to use `yarn infra --help`

Reply:
Just updated to the latter suggestion